product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -457,6 +458,138 @@ witnessGroups:
       maintenanceStartDay: monday
 ```
 
+#### AWS example
+
+```
+clusterName: biganimal-aws-cluster                                      # <string:  cluster name>
+password: password@12345                                              # <string:  cluster password (must be at least 12 characters)>
```

Comment:
lets keep it same across all examples i.e copy from `Azure example`

Reply:
Applied suggested edit